warehouse/query-core/src/main/java/datawave/query/iterator/profile/EvaluationTrackingNestedIterator.java

```diff
@@ -25,6 +25,7 @@ public EvaluationTrackingNestedIterator(QuerySpan.Stage stageName, QuerySpan myS
 
     @Override
     public T next() {
+        mySpan.next();
```

Comment:
@ivakegg just checking if this is still needed.  not sure what the context surrounding this change was.

Reply:
I am pretty sure this is still correct.  This is tracking the next/seek calls in the evaluation pipeline.

Reply:
ok, cool.  I just wasn't familiar with it so wanted to bring it to your attention.